docs/theming/developers/developers.md

```diff
@@ -246,16 +246,40 @@ rectangle.
 
 ```html rhcodeblock
 <svg slot="header" width="80" height="80">
-  <rect fill="var(--rh-color-border-interactive)"
+  <rect fill="var(--rh-color-border-interactive, #0066CC)"
         fill-opacity="0.1"
         stroke-dasharray="4"
         stroke-width="1"
-        stroke="var(--rh-color-border-interactive)"
+        stroke="var(--rh-color-border-interactive, #0066CC)"
         width="80"
         height="80"/>
 </svg>
 ```
 
+When using this approach, there are some important things to keep in mind:
+
+First, all element IDs on the page must be unique, including inline SVG elements.
```

Comment:
Could maybe make this an `<ol>` and nix the "First" and "Second" in each paragraph. Might make it easier to scan.